### PR TITLE
Add a tutorial for production on Winbloat

### PR DESCRIPTION
**Pull request by monitrr**
This PR adds a tutorial for production on Winbloat.

### CODE REVIEW

**Comment by locness3** [comment]
Is npx really needed in  this case? O_o
Also using npx works on Linux, so you can just straight up change the package.json

**Review by Semisol — CHANGES_REQUESTED** [review]
Instead of putting the package.json inside the README, it should be its own file `package.windows.json`, or even better: Use something to detect the environment and run based off of that.

**Comment by locness3** [comment]
> Instead of putting the package.json inside the README, it should be its own file `package.windows.json`, or even better: Use something to detect the environment and run based off of that.

Even better! The modifified package.json should w/o problems on linux *and macOS* ;)

**Comment by Semisol** [comment]
> > Instead of putting the package.json inside the README, it should be its own file `package.windows.json`, or even better: Use something to detect the environment and run based off of that.
> 
> Even better! The modifified package.json should w/o problems on linux _and macOS_ ;)

Yeah

**Review by Semisol — CHANGES_REQUESTED** [review]
Oops, review removed

File: README.md [hunk 1]
```diff
@@ -30,6 +30,74 @@ Then run the build script:
...
+    "vue-jest": "^5.0.0-0"
+  },
+  "description": "A mobile client for Scratch"
+}
+
+  ```
+  </details>
+  
+</details>
+
```
Comment (micahlt) [inline README.md:100]: Unfortunately this is _extremely_ unreliable as dependencies change very often.  I can't merge this with this approach.

**Review by micahlt — CHANGES_REQUESTED** [review]

**Comment by locness3** [comment]
JUST USE npx IN THE REGULAR PACKAGE.JSON

**Review by Semisol — CHANGES_REQUESTED** [review]
Just add the npx script to the package.json, it does not need to be changed

**Pull request closed without merging**